service/templates/projects.html

```diff
@@ -656,20 +657,25 @@ <h3>Virtual Workstations Commands</h3>
             <div class="col-lg-2"> <label class="control-label"> Directory </label> </div>
             <div class="col-lg-2"></div>
           </div>
-
-          <div id="workstation-list" class="form-group" ng-repeat="obj in settingsFormData.workstation_config.setup_commands track by $index">
-            <div class="col-lg-4">
-              <textarea class="form-control" ng-model="obj.command" placeholder="command"></textarea>
-            </div>
-            <div class="col-lg-2">
-              <input class="form-control" ng-model="obj.directory" type="text" placeholder="directory">
-            </div>
-            <div class="col-lg-2">
-              <button class="btn btn-default btn-danger" type="button" ng-click="removeWorkstationCommand($index)">
-                <i class="fa fa-trash"></i>
-              </button>
-            </div>
-          </div>
+          <ol id="workstation-list" class="sortable-list">
+            <li ng-repeat="obj in settingsFormData.workstation_config.setup_commands track by $index">
+              <md-card flex=70>
+                <md-card-content>
```

Comment:
Could you add a style of cursor:move on this white area? It's not clear that these are draggable